js/ui/paygUnlockDialog.js

```diff
@@ -319,13 +319,19 @@ var PaygUnlockDialog = new Lang.Class({
     },
 
     _showError: function(error) {
-        if (error.matches(PaygManager.PaygErrorDomain, PaygManager.PaygError.INVALID_CODE))
+        if (error.matches(PaygManager.PaygErrorDomain, PaygManager.PaygError.INVALID_CODE)) {
             this._setErrorMessage(_("Invalid code"));
-        else if (error.matches(PaygManager.PaygErrorDomain, PaygManager.PaygError.CODE_ALREADY_USED))
+        } else if (error.matches(PaygManager.PaygErrorDomain, PaygManager.PaygError.CODE_ALREADY_USED)) {
             this._setErrorMessage(_("Code already used"));
-        else if (error.matches(PaygManager.PaygErrorDomain, PaygManager.PaygError.TOO_MANY_ATTEMPTS))
+        } else if (error.matches(PaygManager.PaygErrorDomain, PaygManager.PaygError.TOO_MANY_ATTEMPTS)) {
             this._setErrorMessage(_("Too many attempts"));
+        } else {
+            // We don't consider any other error here (and we don't consider DISABLED explicitly,
+            // since that should not happen), but still we need to show something to the user.
+            this._setErrorMessage(_("Unknown Error"));
```

Comment:
Nitpick: The Capitalisation Of The Error Message Is Inconsistent With The Error Messages Above.

Reply:
You. Are. right (hah!)